ui/components/mission/MissionTokenInfo.tsx

```diff
@@ -23,13 +23,13 @@ export default function MissionTokenInfo({
       />
       <p>{`If funding is completed successfully, the locked tokens are treated as follows: `}</p>
       {[
-        '10% of the token is locked indefinitely on an AMM.',
-        "10% of the token is locked for one year and vested for three years, to be held by MoonDAO's Treasury.",
-        '30% of the token is locked for one year, and vested for three years, to be held by the Project Team to distribute how they see fit.',
+        '10% of the token is locked indefinitely on an Automated Market Maker (AMM).',
+        "10% of the token is locked for one year, and vested for three years, to be held by MoonDAO's Treasury.",
+        '30% of the token is locked for one year, and vested for three years, to be held by the Mission Team to distribute how they see fit.',
       ].map((item: string, index: number) => (
         <p key={index} className="ml-1">{`● ${item}`}</p>
       ))}
-      <p>{`If the project does not graduate (their minimum goal was not met), then redemptions are turned on and people can get their full contribution back.`}</p>
+      <p>{`If the project does not launch (their funding goal was not met), then contributors can get their full contribution back. Contributors can also refund their contribution prior to reaching the funding goal.`}</p>
```

Comment:
delete the "can refund prior to reaching funding goal" -- unless if we want to add this as functionality - which we could?